rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.retrievers/10_standard_retriever.yml

```diff
@@ -0,0 +1,514 @@
+          size: 3
+          fields: [ "name", "count" ]
+          retriever:
+            standard:
+              search_after: [ 3 ]
+              query:
+                bool:
+                  should: [
+                    {
+                      term: {
+                        color: "spotted"
+                      }
+                    },
+                    {
+                      term: {
+                        color: "pink"
+                      }
+                    }
+                  ]
+              sort: [
+                {
+                  count: "desc"
+                }
+              ]
+
+  - match: {hits.total.value: 7}
+
+  - match: {hits.hits.0._id: "2"}
+  - match: {hits.hits.0.fields.name.0: "cow cow"}
+  - match: {hits.hits.0.fields.count.0: 2}
+
+---
+"standard retriever terminate after":
+  - do:
+      search:
+        index: animals
+        body:
+          fields: [ "name", "count" ]
+          retriever:
+            standard:
+              filter:
+                bool:
+                  must_not:
+                    match:
+                      name: "cow"
+              sort: [
+                {
+                  count: "asc"
+                }
+              ]
+              terminate_after: 3
+
+  - match: {hits.total.value: 3}
+
+  - match: {hits.hits.0._id: "4"}
+  - match: {hits.hits.0.fields.name.0: "pig"}
+  - match: {hits.hits.0.fields.count.0: 4}
+
+  - match: {hits.hits.1._id: "5"}
+  - match: {hits.hits.1.fields.name.0: "pig pig"}
+  - match: {hits.hits.1.fields.count.0: 5}
+
+  - match: {hits.hits.2._id: "6"}
+  - match: {hits.hits.2.fields.name.0: "pig pig pig"}
+  - match: {hits.hits.2.fields.count.0: 6}
+
+---
+"standard retriever min score":
+  - do:
+      search:
+        index: animals
+        body:
+          fields: [ "name", "count" ]
+          retriever:
+            standard:
+              query:
+                script_score:
+                  query:
+                    match:
+                      name: "cow"
+                  script:
+                    source: " $('count', -1)"
+              min_score: 1.5
+
+  - match: {hits.total.value: 2}
+
+  - match: {hits.hits.0._id: "3"}
+  - match: {hits.hits.0.fields.name.0: "cow cow cow"}
+  - match: {hits.hits.0.fields.count.0: 3}
+
+  - match: {hits.hits.1._id: "2"}
+  - match: {hits.hits.1.fields.name.0: "cow cow"}
+  - match: {hits.hits.1.fields.count.0: 2}
+
+---
+"standard retriever collapse":
+  - do:
+      search:
+        index: animals
+        body:
+          size: 15
+          fields: [ "name", "count" ]
+          retriever:
+            standard:
+              query:
+                match_all: {}
+              collapse:
+                field: "color"
+              sort: [
+                {
+                  count: "asc"
+                }
+              ]
+
+  - match: {hits.total.value: 15}
+
+  - match: {hits.hits.0._id: "1"}
+  - match: {hits.hits.0.fields.name.0: "cow"}
```

Comment:
would be nice to also output "color" field, as as this is the field we are collapsing on.

Reply:
Changed.